test.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import requests
+
```

Comment:
I'd add a docstring in triple quotes at the top of this script to explain what it does and how to use it (even if it's fairly obvious).

e.g.:

```
This script makes an example request to the X endpoint of the service.

Instructions:
1. Run the service: `python main.py`
2. Run this script.
```

Reply:
Done